src/commands/releases.rs

```diff
@@ -170,7 +170,17 @@ pub fn make_app<'a, 'b: 'a>(app: App<'a, 'b>) -> App<'a, 'b> {
             .arg(Arg::with_name("show_projects")
                 .short("P")
                 .long("show-projects")
-                .help("Display the Projects column")))
+                .help("Display the Projects column"))
+            .arg(Arg::with_name("raw")
+                .short("R")
+                .long("raw")
+                .help("Print raw, delimiter separated list of releases. [defaults to new line]"))
+            .arg(Arg::with_name("delimiter")
+                .short("D")
+                .long("delimiter")
+                .takes_value(true)
+                .requires("raw")
+                .help("Delimiter for the --raw flag")))
```

Comment:
Do you think we should set a `default_value` to a `\n`? here instead of unwrapping the option below?

Reply:
No, because then it'd make `--raw` always required per `requires('raw')` is triggered when default value is provided.
Hence this commit: `Use onwrap_or instead of default_value to make it non-required`

Reply:
Added tests

Reply:
Makes sense 👍